### PR TITLE
7.x 4.x basic 725

### PR DESCRIPTION
**Pull request by cynmij**
Added Affiliation term reference field
Deleted Type list field

*not ready to merge yet*

I need to update the person views still

### CODE REVIEW

**Comment by cynmij** [comment]
Update to faculty, staff, students views -- using affiliation field.

**Comment by cynmij** [comment]
Latest commit takes care of https://stanfordits.atlassian.net/browse/BASIC-874 for Person auto node title

**Comment by cynmij** [comment]
Latest commit takes care of https://stanfordits.atlassian.net/browse/BASIC-1180 for Person node display

File: stanford_person.views_default.inc [hunk 1]
```diff
@@ -349,7 +349,7 @@ function stanford_person_views_default_views() {
   $handler->display->display_options['fields']['field_s_person_last_name']['id'] = 'field_s_person_last_name';
   $handler->display->display_options['fields']['field_s_person_last_name']['table'] = 'field_data_field_s_person_last_name';
   $handler->display->display_options['fields']['field_s_person_last_name']['field'] = 'field_s_person_last_name';
-  /* Field: Content: Type */
+  /* Field: Broken/missing handler */
```
Comment (jbickar) [inline stanford_person.views_default.inc]: Why are there now broken/missing handlers in here?

File: stanford_person.views_default.inc [hunk 2]
```diff
@@ -444,7 +444,7 @@ function stanford_person_views_default_views() {
   $handler->display->display_options['filters']['status']['expose']['label'] = 'Published';
   $handler->display->display_options['filters']['status']['expose']['operator'] = 'status_op';
   $handler->display->display_options['filters']['status']['expose']['identifier'] = 'status';
-  /* Filter criterion: Content: Type (field_s_person_type) */
+  /* Filter criterion: Broken/missing handler */
```
Comment (jbickar) [inline stanford_person.views_default.inc]: Another broken/missing handler here.

**Comment by cynmij** [comment]
Manage Content view was the culprit -- fixed now

**Comment by jbickar** [comment]
@sherakama The deleted field (field_s_person_type) was a list field with 3 allowed values. @cynmij confirmed that it is redundant, and should be deleted.

We should do a couple of things as part of this new functionality:

1) Prepopulate the vocabulary with the terms Faculty, Staff, Student. I think this actually should go in this Feature, rather than the JS* install profile(s)
2) In upgrade hook, map the field_s_person_type field value (if it's populated) to the corresponding taxonomy term.

**Comment by sherakama** [comment]
agreed. 
Thanks for following up with the back story on this one.

**Pull request merged**